actian_odbc/manifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='actian_odbc' superclass='odbc' plugin-version='1.0.6' name='Actian Data Platform ODBC' version='18.1'>
```

Comment:
I don't think we released this so I'm not sure this is needed unless this is to keep internal versioning clear?

Reply:
My intent was to move on from release 1.0.6 so that the changes would be under a new internal version. I should have used 1.0.7-dev1 or something else distinct from 1.0.6 and conforming to [Semantic Versioning](https://semver.org/) to make it clearer and consistent.

Reply:
I have updated the version listed in `manifest.xml` to `1.0.7-dev1` for both the JDBC and ODBC connectors.